LESSON2.md

```diff
@@ -361,8 +361,3 @@ stdin.on('end', () => {
 
 *This is exactly the same as our read stream example at the top of this readme, as
 process.openStdin() is a read stream!*
-
-### Project
-
-With what we have covered in these two lessons, we now have everything we need to build some command-line tools! Move onto
```

Comment:
@bradreeder that makes sense, I'm happy to leave it as it was previously - I guess it just needs to be clear to those delivering this (i.e. in the future, all over the world..) what their options are